### DISCUSSION TITLE
Base firmware for bootloader update to 7SEG unit

### DISCUSSION THREAD

**Discussion by realcodfish** [SynthstromAudible/DelugeFirmware#791] [Q&A]
Hi,

The Bootloader Precautions doc recommends I update the bootloader (yes I have a very early one) on my 7SEG Deluge using 4.1.4-beta. Has this firmware arrived anywhere? I see 4.0.1 on the official site at the moment... Or is there an alternative? 

I took a risk with Community V1.0.0 and bootloader 0007 but got the message 'CAnt' so I guess bootloader updates may have been disabled somehow.

**Comment by m-m-adams** [comment]
Are you doing dev work? If you just want to use the firmware the old bootloader still works fine

**Comment by ok-reza** [comment]
to your questions:
4.1.4-beta (latest is v4.1.4-RC) is available on the official synthstrom forums but you have to approved to get access: https://forums.synthstrom.com/discussion/3443/how-to-be-involved-in-beta-testing#latest

v4.1.4 is in release candidate stage, meaning its release is imminent, it should be officially released in a couple weeks if you'd prefer to wait.

and yes, @m-m-adams is right that you can still use community firmware with the old bootloader, the risks are likely rare, for 6 months i've had bootloader 0005 and have swapped firmware countless of times with no issues. but i also understand, especially with expensive gear, that peace of mind is worth a lot too.

**Comment by darenager** [comment]
@realcodfish did you solve this yet? If not the “CAnt’ message appears when there is another .bin file on the root of the card, you have to remove any other bin files and put the boot loader.bin in the root on its own, once you have flashed the boot loader you can then remove the card, remove the boot loader .bin and add the firmware .bin then update the firmware.
